templates/miq-template-ext-db.yaml

```diff
@@ -192,6 +192,108 @@ objects:
           resources:
             requests:
               storage: "${APPLICATION_VOLUME_CAPACITY}"
+
+
+- apiVersion: apps/v1beta1
+  kind: "StatefulSet"
+  metadata:
+    name: ${NAME}-backend
+    annotations:
+      description: "Defines how to deploy the ManageIQ appliance"
+  spec:
+    serviceName: "${NAME}-backend"
+    replicas: 0
+    template:
+      metadata:
+        labels:
+          name: ${NAME}-backend
+        name: ${NAME}-backend
+      spec:
+        containers:
+        - name: manageiq
+          image: "${APPLICATION_IMG_NAME}:${BACKEND_APPLICATION_IMG_TAG}"
+          imagePullPolicy: Always
+          livenessProbe:
+            exec:
+              command:
+                - pidof
+                - "MIQ Server"
+            initialDelaySeconds: 480
+          volumeMounts:
+              -
+                name: "${NAME}-server"
+                mountPath: "/persistent"
+          env:
+            -
+              name: "APPLICATION_INIT_DELAY"
+              value: "${APPLICATION_INIT_DELAY}"
+            -
+              name: "DATABASE_SERVICE_NAME"
+              value: "${DATABASE_SERVICE_NAME}"
+            -
+              name: "DATABASE_REGION"
+              value: "${DATABASE_REGION}"
```

Comment:
If this pod is never going to initialize the region I don't think we need to provide this. I think the region id will be detected using the existing database sequence.